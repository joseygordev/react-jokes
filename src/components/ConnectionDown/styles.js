import theme from '../../config/theme';

export default {
  root: {
    width: '100%',
    height: 60,
    backgroundColor: 'red',
    top: 0,
    left: 0,
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  },
  message: {
    fontSize: '1rem',
    color: theme.colors.white,
    textAlign: 'center',
    fontWeight: 'bold',
    padding: theme.spacing.padding * 1.4
  }
};
