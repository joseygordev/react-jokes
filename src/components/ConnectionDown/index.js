import React from 'react';

// Styles
import styles from './styles';

const ConnectionDown = () => {
  return (
    <div style={styles.root}>
      <p style={styles.message}>You are disconnected, we are doing our best to keep your experience pleasant :)</p>
    </div>
  );
};

export default ConnectionDown;
