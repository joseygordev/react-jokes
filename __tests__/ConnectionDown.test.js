import React from 'react';
import renderer from 'react-test-renderer';
import ConnectionDown from '../src/components/ConnectionDown';

describe('ConnectionDown', () => {
  it('Should return the ConnectionDown component', () => {
    const tree = renderer
      .create(<ConnectionDown />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
