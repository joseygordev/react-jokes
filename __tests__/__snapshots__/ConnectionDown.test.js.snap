// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectionDown Should return the ConnectionDown component 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "red",
      "display": "flex",
      "height": 60,
      "justifyContent": "center",
      "left": 0,
      "top": 0,
      "width": "100%",
    }
  }
>
  <p
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": "1rem",
        "fontWeight": "bold",
        "padding": 11.2,
        "textAlign": "center",
      }
    }
  >
    You are disconnected, we are doing our best to keep your experience pleasant :)
  </p>
</div>
`;
